_schedule/talks/2021-10-22-modern-javascript-for-django-developers.md

```diff
@@ -44,18 +44,18 @@ video_url: ''
 
 Is the front end of your Django project a disorganized mess? Have you ever tried to add React to a Django project and gotten discouraged because it was too complicated? Or have you worked in a project with completely separate front and back ends and wondered why you couldn't use just Django templates and forms for simple things and had to build an API for every little thing?
 
-If so, this talk is for you. It will include three high-level parts.
+If so, this talk is for you.
 
-Part 1 is a discussion of common Django / JavaScript architectures. These include:
+First I cover common Django / JavaScript architectures and their tradeoffs. Including:
 
-1. The most common "ad-hoc" approach, where JS just gets incrementally added to templates until you find yourself in a mess
-2. The "completely decoupled" JS front-end and Django-API back end, which is popular, but complex and unfamiliar to pure Django developers.
+1. The most common "ad-hoc" approach, where JS just gets incrementally added to templates until you find yourself in a mess.
+2. The completely decoupled JS front-end and Django-API back end, which is popular, but complex and unfamiliar to Django developers.
 3. A "hybrid architecture", that combines modern JS with familiarity of working in Django.
 
-The rest of the talk will deep dive into building out a hybrid app with Django.
+The rest of the talk deep dives into building out a hybrid app with Django.
 
-Part 2 will focus on the fundamentals of JS tooling - a prerequisite to working with modern JavaScript. I'll start with explaining why it's so frustrating and confusing trying to add React to a Django project. Then introduce the concept of a JavaScript toolchain. Why you need them and what they do. And finally do a quick overview of the most common JS toolchain: NPM, Webpack, and Babel, including what each does and the analogies in the Python world.
+I'll start with the fundamentals of JS tooling - a prerequisite to working with modern JavaScript. I'll cover why it's so frustrating and confusing trying to add React to a Django project, then introduce the concept of a JavaScript toolchain, why you need them and what they do, and finally do a quick overview of the most common JS toolchain: NPM, Webpack, and Babel.
```

Comment:
Grammar question:

Since you're covering Django/JS archs and tradeoff before you talk about tooling, should that maybe say "Next, I'll cover the fundamentals of JS tooling"?

Reply:
Good point. I tweaked this a bit to make it clear the "start" only applies to the section.